README.md

```diff
@@ -54,6 +55,94 @@ There are a number of other resources you can review to learn to use this plugin
 * [snap mysql integration test](https://github.com/intelsdi-x/snap-plugin-publisher-mysql/blob/master/mysql/mysql_integration_test.go)
 * [snap mysql unit test](https://github.com/intelsdi-x/snap-plugin-publisher-mysql/blob/master/mysql/mysql_test.go)
 
+### Task Manifest Config
+
+In task manifest, the config section of mysql publisher describes how to establish a connection to the MySQL server.
+
+Name 	  	 | Data Type | Default       | Description
+----------|-----------|---------------|-------------
+hostname 	| string 	  | localhost     | the host of MySQL service
+port 		   | string	 	 | 3306          | the port number of MySQL service
+username  | string 	  | root          | the name of user
+password 	| string 	  | root          | the password of user
+database 	| string 	  | SNAP_TEST        | the name of database (use existed or create a new)
+tablename | string 	  | info       | the name of table (use existed or create a new)
+
+Each connection parameter has a default value, but it can be override by set a value in task manifest (see [exemplary task manifest](examples/tasks/mock-mysql.json))
```

Comment:
It's worth documenting the default schema for generated metric table. Attach outcome of `DESCRIBE TABLE`.